rules.mk

```diff
@@ -276,7 +276,9 @@ BASH:=bash
 TAR:=tar
 FIND:=find
 PATCH:=patch
-PYTHON:=python
+PYTHON:=python2
+PYTHON2:=python2
```

Comment:
This variable is not needed, python2 is implicit.

Reply:
My idea was to make it explicit and to stop using PYTHON var